### PR TITLE
Add max_n_attemps and block_time parameters to api configuration

### PR DESCRIPTION
**Pull request by CarlosRS9**
Hello team,

This PR updates the `user-manual/api/configuration` documentation page to include the new `max_n_attempts` and `block_time` parameters for the security.yaml configuration file added in [#5318](https://github.com/wazuh/wazuh/issues/5318)

Regards.

### CODE REVIEW

File: source/user-manual/api/configuration.rst [hunk 1]
```diff
@@ -268,3 +272,19 @@ rbac_mode
 +================+===============+================================================================================================================================================================+
 | black,white    | black         | Sets the behavior of RBAC. For example, in black mode, policies not included in the list **can be** executed, while in white mode they **cannot** be executed. |
 +----------------+---------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+max_n_attemps
```
Comment (AdriiiPRodri) [inline source/user-manual/api/configuration.rst:276]: max_n_attempts

File: source/user-manual/api/configuration.rst [hunk 2]
```diff
@@ -268,3 +272,19 @@ rbac_mode
...
+^^^^^^^^^^^^^
++-----------------------+---------------+------------------------------------------------------------------------------------------------------------------------+
+| Allowed values        | Default value | Description                                                                                                            |
++=======================+===============+========================================================================================================================+
+| Any positive integer. | 5             | Set how many failed access attempts are allowed during the time specified with ``block_time`` before blocking that IP. |
```
Comment (AdriiiPRodri) [inline source/user-manual/api/configuration.rst:281]: Zero not included

File: source/user-manual/api/configuration.rst [hunk 3]
```diff
@@ -268,3 +272,19 @@ rbac_mode
...
++-----------------------+---------------+------------------------------------------------------------------------------------------------------------------------+
+
+block_time
+^^^^^^^^^^
++-----------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| Allowed values        | Default value | Description                                                                                                                       |
++=======================+===============+===================================================================================================================================+
+| Any positive integer. | 300           | Set how many seconds it takes for the ``max_n_attemps`` to expire and start counting again the number of access attemps received. |
```
Comment (AdriiiPRodri) [inline source/user-manual/api/configuration.rst:289]: Zero included -> 0 = Blocking system disabled

File: source/user-manual/api/configuration.rst [hunk 4]
```diff
@@ -253,6 +255,8 @@ cache
 Security configuration options
 ------------------------------
 
+This section contains all the options available for the ``/var/ossec/api/security/security.yaml`` configuration file.
```
Comment (AdriiiPRodri) [inline source/user-manual/api/configuration.rst:258]: The correct path is: {WAZUH_PATH}/api/configuration/security/security.yaml

File: source/user-manual/api/configuration.rst [hunk 5]
```diff
@@ -150,6 +150,8 @@ And remove temporary files:
 API configuration options
 -------------------------
 
+This section contains all the options available for the ``/var/ossec/api/configuration/api.yaml`` configuration file.
```
Comment (AdriiiPRodri) [inline source/user-manual/api/configuration.rst:153]: Change /var/ossec for {WAZUH_PATH}

Comment (CarlosRS9) [reply source/user-manual/api/configuration.rst]: Done

File: source/user-manual/api/configuration.rst (hunk 4 above)
Comment (CarlosRS9) [reply source/user-manual/api/configuration.rst]: Done

File: source/user-manual/api/configuration.rst (hunk 3 above)
Comment (CarlosRS9) [reply source/user-manual/api/configuration.rst]: Done

File: source/user-manual/api/configuration.rst (hunk 2 above)
Comment (CarlosRS9) [reply source/user-manual/api/configuration.rst]: Done

File: source/user-manual/api/configuration.rst (hunk 1 above)
Comment (CarlosRS9) [reply source/user-manual/api/configuration.rst]: Done

**Comment by davidjiglesias** [comment]
Already done https://github.com/wazuh/wazuh-documentation/pull/2722

**Pull request closed without merging**